lib/slack/web/faraday/response/raise_error.rb

```diff
@@ -15,6 +15,14 @@ def on_complete(env)
             error_class ||= Slack::Web::Api::Errors::SlackError
             raise error_class.new(error_message, env.response)
           end
+
+          def call(env)
+            super
+          rescue Slack::Web::Api::Errors::SlackError, Slack::Web::Api::Errors::TooManyRequestsError => e
```

Comment:
I think `TooManyRequestsError` is an `SlackError` so both here would get rescued with just `SlackError`

Reply:
```
> Slack::Web::Api::Errors::TooManyRequestsError <  Slack::Web::Api::Errors::SlackError
=> nil
```

Reply:
Ah yes because we inherited this one from `Faraday::Error` to get `retry_after`.

Reply:
Maybe there needs to be an intermediate `Slack::Web::Api::Errors::HttpRequestError` that holds an inner `FaradayError` exception?